src/toolbox.css

```diff
@@ -12,32 +12,47 @@ form {
   padding: 1rem;
 }
 
+input {
+  background-color: var(--theme-toolbar-background);
+  border: 1px solid var(--theme-splitter-color);
+  color: var(--theme-body-color);
+  font-size: 1rem;
+  padding: 0.25rem;
+  line-height: 2rem;
+  padding-left: 1rem;
+  display: block;
+  width: 100%;
+}
+
+::-webkit-input-placeholder {
+  color: var(--theme-body-color);
+  font-size: 1rem;
+  line-height: 2rem;
+  margin: 0;
+}
+
+input:-webkit-autofill {
```

Comment:
sure - it's needed for chrome when you select one of the previous options :)

so silly